modules/javafx.graphics/src/main/java/com/sun/javafx/font/PrismFontFactory.java

```diff
@@ -1367,15 +1367,9 @@ public PGFont[] loadEmbeddedFont(String name, InputStream fontStream,
                                      float size,
                                      boolean register,
                                      boolean loadAll) {
-        if (!hasPermission()) {
-            return new PGFont[] { createFont(DEFAULT_FULLNAME, size) } ;
-        }
-        if (FontFileWriter.hasTempPermission()) {
-            return loadEmbeddedFont0(name, fontStream, size, register, loadAll);
-        }
```

Comment:
This test always returns true today, so it looks like you removed the code path that was in use. Unless I'm missing something, I think that you should restore the "return loadEmbeddedFont0" call and remove everything else below this point from this method.

Reply:
Yes, this should always `return loadEmbeddedFont0`. Again, some Friday-related mishap.

Reply:
I went a step further even - now with `loadEmbeddedFont` essentially only calling `loadEmbeddedFont0` this is the only place where `loadEmbeddedFont0` is used. I decided to remove this code altogether and rename `loadEmbeddedFont0` to appropriate `loadEmbeddedFont`. Tests seem to be all good after this change.